endpoints/api/superuser.py

```diff
@@ -204,13 +204,32 @@ class SuperUserOrganizationList(ApiResource):
     @require_fresh_login
     @verify_not_prod
     @nickname("listAllOrganizations")
+    @parse_args()
+    @query_param(
+        "all",
+        "Returns all organizations within the registry with no pagination",
+        type=truthy_bool,
+        default=True,
+    )
```

Comment:
I don't think we need this anymore. The caller should just go through every pages until no token is returned.

Reply:
I'm thinking we may want this for backwards compatibility. Is it possible user's have written automation that expect all orgs to be returned?

Reply:
updated